docs/project_governance.rst

```diff
@@ -47,39 +140,67 @@ Decision Process
-   * In case of a tie, the decision on the proposal is taken by the Committee Chair.
+   Majority vote is a mechanism to override an obstructing veto. However, it is the intention of the QAX
+   collaborators that the SC should always strive to reach a consensus and that a majority vote should only be
+   enacted once all other resolution options are exhausted, and the SC reach an impasse.
+
+   * If a majority vote is called the proposer and the vetoer must both make their case before the SC.
+   * All SC members must vote (using any score options outlined above). For a proposal to pass a majority
+     vote it must receive a total score of at least +1.
+   * In case of a tie (final score is 0), the decision on the proposal is taken by the Committee Chair.
    * The majority vote can be used to override an obstructionist veto, but it is intended that in normal circumstances
-vetoers need to be convinced to withdraw their veto. We are trying to reach consensus.
+     vetoers need to be convinced to withdraw their veto. We are trying to reach consensus.
 
-When is a Proposal Required?
-----------------------------
 
-Anything that might be controversial. For instance:
+Ad Hoc Meetings
+^^^^^^^^^^^^^^^
 
-* When a relevant release should take place (not required for bug-fix releases).
-* Adding a substantial amount of new code.
-* Changes that breaks backward compatibility.
+Ad Hoc meetings can be called by any SC member and do not need to contain the full complement of QSC
+members. These meetings are to be held to facilitate project work and activities.
+
+At a minimum actions and notes should be emailed to SC following an ad hoc meeting and stored in the
+`wiki associated with this repository <https://github.com/ausseabed/qax/wiki>`_ (??).
 
-Code Changes
-------------
 
-Whether or not a proposal is required, all not-minor code changes require the submission of
+Quality Control
+^^^^^^^^^^^^^^^
+
+Whether or not a proposal is required, all non-minor code changes require the submission of
 a `GitHub pull request <https://github.com/ausseabed/qax/pulls>`_.
 
 Before a proposal is merged, at least 1 reviewer (different than the submitter) needs to approve
 the code changes.
 
-Bootstrapping
--------------
 
-Lachlan Hurst (FrontiersSI), Xxxx Yyyyyy (Geoscience Australia), Zzzz Wwwww (Kkkkk), Tyanne Faulkes (NOAA, OCS) and
-Giuseppe Masetti (UNH, CCOM) are declared to be the founding QAX Steering Committee.
+Risk management
+^^^^^^^^^^^^^^^
 
-Lachlan Hurst is declared initial Chair of the QAX Steering Committee.
+Major proposals need to identify risks associated with work and implement management strategies.
+
+
+Communications and Outreach
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Any activity that has the potential to generate public interest should be discussed as an outreach
+opportunity during the proposal phase.
+
+Where SC agree that public communication is required, the proposer should generate a communication
+plan for the SC to review.
```

Comment:
I am unsure what it means in practice. Can somebody make me a couple of concrete examples?